jung-samples/src/main/java/edu/uci/ics/jung/samples/SpatialLensDemo.java

```diff
@@ -97,25 +93,11 @@ public SpatialLensDemo() {
     final VisualizationModel<String, Number, Point2D> visualizationModel =
         new BaseVisualizationModel<>(graph, graphLayoutAlgorithm, preferredSize);
     vv = new VisualizationViewer<>(visualizationModel, preferredSize);
-
-    PickedState<String> ps = vv.getPickedVertexState();
-    PickedState<Number> pes = vv.getPickedEdgeState();
-    vv.getRenderContext()
-        .setVertexFillPaintTransformer(
-            new PickableVertexPaintTransformer<>(ps, Color.red, Color.yellow));
-    vv.getRenderContext()
-        .setEdgeDrawPaintTransformer(
-            new PickableEdgePaintTransformer<>(pes, Color.black, Color.cyan));
+    vv.getRenderContext().setVertexLabelTransformer(Object::toString);
```

Comment:
I removed these programmatic settings from this particular demo because I want to use properties or defaults. I think these statements just set what was already the default.